netrd/distance/communicability_jsd.py

```diff
@@ -0,0 +1,79 @@
+"""
+communicability_jsd.py
+--------------------------
+
+Distance measure based on the Jensen-Shannon Divergence 
+between the communicability sequence of two graphs as 
+defined in:
+
+Chen, D., Shi, D. D., Qin, M., Xu, S. M., & Pan, G. J. (2018). 
+Complex network comparison based on communicability 
+sequence entropy. Physical Review E, 98(1), 012319.
+
+https://journals.aps.org/pre/abstract/10.1103/PhysRevE.98.012319
+
+author: Brennan Klein
+email: brennanjamesklein@gmail.com
+Submitted as part of the 2019 NetSI Collabathon.
+
+"""
+
+import networkx as nx
+import numpy as np
+import scipy as sp
+from .base import BaseDistance
+
+class CommunicabilityJSD(BaseDistance):
+    def dist(self, G1, G2):
+        """        
+        Params
+        ------
+        G1 (nx.Graph): the first graph
+        G2 (nx.Graph): the second graph
+        
+        Returns
+        -------
+        dist (float): between zero and one, this is the communicability 
+                      sequence distance bewtween G1 and G2.
+        
+        """
+
+        N1 = G1.number_of_nodes()
+        N2 = G2.number_of_nodes()
+
+        C1 = nx.communicability_exp(G1)
+        C2 = nx.communicability_exp(G2)
+
+        Ca1 = np.zeros((N1, N1))
+        Ca2 = np.zeros((N2, N2))
+
+        for i in range(Ca1.shape[0]):
+            Ca1[i] = np.array(list(C1[i].values()))
```

Comment:
I think this is safe since we've already implicitly assumed integer-labeled nodes by doing the for loop in this way, but is it possible that the dicts do not always have the same order?